bin/cli.mjs

```diff
@@ -90,11 +96,13 @@ async function main() {
     // For some silly reason, the OS X open command ignores any query parameters or hash parameters
     // passed as part of the URL. To get around this weird issue, we'll create a local HTML file
     // that just redirects.
-    const htmlPath = path.join(os.tmpdir(), `${filePrefix}.html`)
-    console.log(`Creating temp file ${htmlPath}`)
-    fs.writeFileSync(htmlPath, `<script>window.location=${JSON.stringify(urlToOpen)}</script>`)
+    if (process.platform === 'darwin') {
+      const htmlPath = path.join(os.tmpdir(), `${filePrefix}.html`)
+      console.log(`Creating temp file ${htmlPath}`)
+      fs.writeFileSync(htmlPath, `<script>window.location=${JSON.stringify(urlToOpen)}</script>`)
 
-    urlToOpen = `file://${htmlPath}`
+      urlToOpen = `file://${htmlPath}`
+    }
   }
```

Comment:
I've only actually tested this on OS X, so it's possible this issue exists for other platforms too.

To minimize the possibility of breakage, can we maintain this without the `process.platform` check? If you'd like to avoid this step on the specific platform you're using, I'm also open to skipping this step on any platforms you can test.

Reply:
Open supports 3 platforms: `win32`, `darwin`, and … `else`, so it tries to use `xdg-open` on all platforms that aren’t windows or macOS.

I tried on Windows, and indeed, the hash was also stripped, so I left it only for the platforms using `xdg-open`.